src/docs/deployment/web-renderers.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Web renderers
+description: How to choose a web renderer for running and building a web app.
+---
+
```

Comment:
Add a note stating that this is available on which channels. (NOT STABLE yet)

Reply:
added the note from flutter.dev/web